task5/src/components/movieList/movieList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './movieList.scss';
+import Movie from '../movie/movie';
+import { movies } from '../../mock_data/movies';
+
+function MovieList() {
+  return (
+    <div className="movie-list">
+      <span className="movies-count"><b>{movies.length}</b> Movies found</span>
+      <div className="movie-container">
+        { movies.map(i => (<Movie key={`mov${i.id}`} movie={i}/>)) }
```

Comment:
maybe a more descriptive name for the parameter of the arrow function like movie or something, as we use `i` in for loops like an integer, would be a good help.